package.json

```diff
@@ -695,6 +727,12 @@
         "key": "ctrl+shift+e",
         "mac": "cmd+shift+e",
         "when": "(editorLangId == sqlpp || resourceFilename =~ /.sqlpp$/) && !isKVCluster"
+      },
+      {
+        "command": "vscode-couchbase.runSearch",
+        "key": "ctrl+shift+e",
+        "mac": "cmd+shift+e",
+        "when": "(editorLangId == cbs.json || resourceFilename =~ /.cbs.json$/) && !isKVCluster"
```

Comment:
We will have to add another condition stating isSearchService. Because by default, search service is not mandatory